code/datums/actions/mobs/sneak.dm

```diff
@@ -20,13 +22,13 @@
 	if(HAS_TRAIT(owner, TRAIT_SNEAK))
 		// It's safest to go to the initial alpha of the mob.
 		// Otherwise we get permanent invisbility exploits.
-		owner.alpha = initial(owner.alpha)
-		to_chat(owner, span_noticealien("You reveal yourself!"))
+		animate(owner, alpha = initial(owner.alpha), time = animation_time)
+		owner.balloon_alert(owner, "you reveal yourself!")
 		REMOVE_TRAIT(owner, TRAIT_SNEAK, name)
 
 	else
-		owner.alpha = sneak_alpha
-		to_chat(owner, span_noticealien("You blend into the enviorment..."))
+		animate(owner, alpha = sneak_alpha, time = animation_time)
+		owner.balloon_alert(owner, "you blend into the environment...")
```

Comment:
```suggestion
		owner.balloon_alert(owner, "you blend into the environment")
```
The shorter the message is, the better. Ellipsis are better used for pending states.